test/support/helpers/company-contact.helper.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+/**
+ * @module CompanyContactHelper
+ */
+
+const CompanyContactModel = require('../../../app/models/company-contact.model.js')
+const { generateUUID } = require('../../../app/lib/general.lib.js')
+/**
```

Comment:
NIT - There had to be one to prove I looked at the thing!

```suggestion
const { generateUUID } = require('../../../app/lib/general.lib.js')

/**
```